doc/source/whatsnew/v0.23.1.txt

```diff
@@ -82,7 +82,7 @@ Plotting
 Reshaping
 ^^^^^^^^^
 
--
+- Bug in :func:`concat` where error was raised in concatenating Series with numpy scalar and tuple names (:issue:`21015`)
```

Comment:
double backticks around Series, is this *only* a numpy scalar or any scalar?

Reply:
only numpy scalar - as it returns an array on comparison with tuple

Reply:
ok that's fine (but add in an integer in the test as well below)